calblueprint/meepanyar
# Camera Functionality

[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR
Added Camera and Preview screens to the PR. They're not styled correctly because I can't access them on phone without deploying to a secure connection. Styling will happen in a future PR.

For those wanting to integrate the screen, they must provide a `photo` prop. Directions for use are located in the comments of `Camera.tsx`.

This PR needs to be run with the associated backend [PR](https://github.com/calblueprint/meepanyar-node/pull/13)

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## How to review
1. Run the appropriate backend branch after asking Julian for the environment variables
2. Go to `/confirm-payment` endpoint and click the "Add Photo" section to be brought to the `/camera` screen
3. Take a picture. The `/camera/preview` endpoint uploads the picture after you press "Next" for demonstration purposes
4. Check the "Financial Summaries" table of Airtable to make sure your photo was properly persisted.

[//]: # 'The order in which to review files and what to expect when testing locally'

## Relevant Links

### Online sources

[//]: # 'Optional - copy links to any tutorial or documentation that was useful to you when working on this PR'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## Next steps
- Figure out how to style the camera. I'd like to see in prod before attempting to style.
[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

### Screenshots

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @julianrkung

[//]: # 'This tags Julian as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
